src/main.qml

```diff
@@ -171,29 +171,28 @@ Application {
                 }
 
                 Rectangle {
-                    id: minMaxCircle
+                    id: minCircle
 
-                    width: Dims.w(26)
+                    width: Dims.w(30)
                     height: width
                     radius: width/2
                     color: "#00ffffff"
                     anchors {
                         centerIn: parent
-                        horizontalCenterOffset: Dims.w(32)
+                        horizontalCenterOffset: Dims.w(-30)
                     }
 
                     Label {
                         id: minDisplay
 
-                        width: Dims.w(40)
+                        width: parent.width
```

Comment:
Good solution to depend the size on the parent.
Using Dims here would have broken the centering when displaying on screens with unusual width/height ratio.